### ISSUE TITLE
Multi Camera Multi Target Python* Demo Code - OrientationFeature Class

### ISSUE THREAD

**Issue by Carrimel** [openvinotoolkit/open_model_zoo#2240]
Hi,

I see that the OrientationFeature class is defined in `mc_tracker/sct.py` and there are attributes for it in the Track class. However, by default it never ends up being used (default constructors all have None). Could someone explain what the purpose of this class is and if it is meant to be used then what is the correct way to take advantage of it?

**Comment by vladimir-dudnik** [comment]
@sovrasov could yo please comment on this?

**Comment by sovrasov** [comment]
Hi @Carrimel. Originally we tried to build our tracker based on approach described [here](https://arxiv.org/pdf/1906.01357.pdf). 
This paper explains the concept of this feature. We tried to avoid usage of a resource consuming pose estimation model and train a small classifier to directly predict orientation (left, righr, front, back). Due to lack of the annotated data we were unable to reach a reasonable prediction quality, so we didn't publish the orientation classifier and don't actually use the OrientationFeature in the tracking pipeline.
